docker-for-mac/release-notes.md

```diff
@@ -16,10 +16,40 @@ This page contains information about the new features, improvements, known issue
 > Starting with Docker Desktop 3.0.0, Stable and Edge releases are combined into a single release stream for all users. Updates to Docker Desktop will now be available automatically as delta updates from the previous version. This means, when there is a newer version of Docker Desktop, it will be automatically downloaded to your machine. All you need to do is to click **Update and restart** from the Docker menu to install the latest update.
 {: .important }
 
+## Docker Desktop 3.3.0
+2021-04-08
+
+> [Download](https://desktop.docker.com/mac/stable/amd64/Docker.dmg)
+
+### Bug fixes and minor changes
+
+- Fixed an issue related to docker-compose action. Fixes [docker/for-win#10564](https://github.com/docker/for-win/issues/10564).
+- Fixed launching Docker Desktop from the Dock. Fixes [docker/for-mac#5515](https://github.com/docker/for-mac/issues/5515).
+- Ensure `--add-host host.docker.internal:host-gateway` causes `host.docker.internal` resolves to the host IP, rather than the IP of the IP router. See [docker/for-linux#264](https://github.com/docker/for-linux/issues/264).
+- Prevent inter-container HTTP traffic being forwarded via the HTTP proxy to the host. Fixes [docker/for-mac#5476](https://github.com/docker/for-mac/issues/5476).
+- Prevent the deletion of files which have been added to the folder containing the VM disk image. Fixes [docker/for-mac#5486](https://github.com/docker/for-mac/issues/5486).
+- Fixed an issue where delta downloads caused an illegal instruction exception. Fixes [docker/for-mac#5459](https://github.com/docker/for-mac/issues/5459).
```

Comment:
```suggestion
- Fixed an issue where delta downloads caused an `Illegal instruction exception` . Fixes [docker/for-mac#5459](https://github.com/docker/for-mac/issues/5459).
```

Reply:
Was thinking of using a different term instead of 'Illegal', but then the realized it is the actual error message. Suggest wrapping it in quotes for emphasis.